_includes/docs/user-guide/contribution/widgets-development.md

```diff
@@ -815,6 +880,498 @@ Only custom **showAlert** function was implemented showing an alert with the con
+                    decimals: 0,
+                    label: "Active Devices",
+                    name: "count",
+                    settings: {},
+                    type: "count"
+                }
+            ],
+            entityFilter:
+            {
+                type: "entityType",
+                entityType: "DEVICE"
+            },
+            keyFilters: //Filtering entity by keys
+            [
+                {
+                    key: {
+                        key: "active", //Key name
+                        type: "ATTRIBUTE" //Key type
+                    },
+                    predicate: {
+                        operation: "EQUAL", //Operation type
+                        type: "BOOLEAN", //Sorting value type
+                        value: {
+                            defaultValue: true //Sorting value
+                        }
+                    },
+                    valueType: "BOOLEAN" //Value type
+                }
+            ]
+        }
+    ];
+
+    const options = {
+        type: 'latest', //Subscription type
+        datasources: datasources, //Describes what data you want to subscribe
+        callbacks: //Sets callbacks for subscription
+        {
+            onDataUpdated: () => {
+                self.onDataUpdated();
+            }
+        }
+    };
+    
+    self.ctx.subscriptionApi.createSubscription(subscriptionOptions, true).subscribe(
+        (subscription) => {
+            self.ctx.defaultSubscription = subscription; //Saving subscription information into widget context
+            self.ctx.data = subscription.data; //Saving data into widget context
+            self.ctx.datasources = subscription.datasources; //Saving datasource into widget context
+            ...
+        }
+    );
+    ...
+}
+...
+```
+
+As a result, will be created subscription to count devices in the system and count active devices (**the widget is illustrative**):
+![image](/images/user-guide/contribution/widgets/count-subscription.png)
+
+
+##### Subscription for attributes/telemetry
+
+Let's create a custom subscription to the latest **temperature** key value for active devices:
+```javascript
+self.onInit = function() {
+    ...
+    const datasources = [
+        {
+            type: "entity", //Sets that there is a subscription to entity data
+            dataKeys: //Describes keys
+            [
+                {
+                    decimals: 0, //Number of digits after floating point for this key
+                    label: "Temperature", //Key label
+                    name: "temperature", //Key name
+                    settings: {},
+                    type: "timeseries" //Key type
+                },
+                {
+                    decimals: 0,
+                    label: "Active",
+                    name: "active",
+                    settings: {},
+                    type: "attribute"
+                 }
+            ],
+            entityFilter: //Describes entities
```

Comment:
please add link to all available entityFilters

Reply:
Link to the interface EntityFilter already added in datasource fields description.
![image](https://github.com/thingsboard/thingsboard.github.io/assets/43555187/18bc2275-5d8c-4fb7-afd6-f771473bb54e)